src/components/tuits/tuit.js

```diff
@@ -47,14 +47,18 @@ const Tuit = ({tuit, deleteTuit, likeTuit, dislikeTuit}) => {
                     <i onClick={() => deleteTuit(tuit._id)}
                        className="fas fa-remove tuit-button fa-2x fa-pull-right"/>
                 }
-                <Link to={`/tuit/${tuit._id}`}>
+                { tuit.ownedByMe === true &&
+                    <Link to={`/tuit/${tuit._id}`}>
                     <i className="float-end tuit-button fas fa-circle-ellipsis me-1"/>
                 </Link>
-                <h2
-                    className="fs-5">
-                    {tuit.postedBy && tuit.postedBy.username}
-                    @{tuit.postedBy && tuit.postedBy.username} -<span className="ms-1">{daysOld(tuit)}</span> </h2>
-                {tuit.tuit}
+                }
+                <div>
+                <Link
```

Comment:
The tuits in the admin page's "Tuits List" should also have the link to another user's profile as well

Reply:
I believe the tuit component should be fairly similar regardless of where it is shown. Thoughts @caijiah ?